### ISSUE TITLE
Where are timestamps?

### ISSUE THREAD

**Issue by stricte** [documentcloud/jammit#268]
Am i missing sth or change committed in dcc9b02c2b9254fc4e6e53ba4a2dbc36378540cd left us without timestamps in generated urls?

**Comment by knowtheory** [comment]
Yep, seems like a problem.  Pull requests welcome, otherwise we'll get to it as we're updating the rails integration.

**Comment by knowtheory** [comment]
Apologies looks like #263 covers it, gonna review that.
